examples/dmd/wave_equation.cpp

```diff
@@ -0,0 +1,685 @@
+ /*
+ * Copyright (c) 2013-2022, Lawrence Livermore National Security, LLC
+ * and other libROM project developers. See the top-level COPYRIGHT
+ * file for details.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR MIT)
+ *
+ *****************************************************************************/
+//
+//                       libROM MFEM Example: Wave_Equation (adapted from ex23.cpp)
+//
+// Compile with: make wave_equation
+//
+// Sample runs:  wave_equation
+//               wave_equation -o 4 -tf 5
+//               wave_equation -m ../data/square-disc.mesh -o 2 -tf 2 --neumann
+//               wave_equation -m ../data/disc-nurbs.mesh -r 3 -o 4 -tf 2
+//               wave_equation -m ../data/inline-hex.mesh -o 1 -tf 2 --neumann
+//               wave_equation -m ../data/inline-tet.mesh -o 1 -tf 2 --neumann
+//
+// =================================================================================
+//
+// Sample runs and results for DMD:
+//
+// Command 1:
+//   wave_equation -o 4 -tf 5
+//
+// Output 1:
+//   Relative error of DMD temperature (u) at t_final: 5.0 is ??
+//
```

Comment:
Also add computational time info as well.

Reply:
One info with -visit option and another info without -visit option.

Reply:
Done.